src/sidebar/components/annotation-share-control.js

```diff
@@ -141,14 +143,23 @@ function AnnotationShareControl({
                 className="annotation-share-panel__icon-button"
               />
             </div>
-            <div className="annotation-share-panel__permissions">
-              {annotationSharingInfo}
-            </div>
-            <ShareLinks
-              shareURI={shareUri}
-              analyticsEventName={analytics.events.ANNOTATION_SHARED}
-              className="annotation-share-control__links"
-            />
+            {inContextAvailable ? (
+              <div className="annotation-share-panel__details">
+                {annotationSharingInfo}
+              </div>
+            ) : (
+              <div className="annotation-share-panel__details">
+                This annotation cannot be shared in its original context because
+                it was made on a document that is not available on the web. This
+                link shares the annotation by itself.
+              </div>
+            )}
+            {inContextAvailable && (
```

Comment:
In the case where only the non-incontext link is available, the social media sharing options could still share that.

Reply:
We don't have the vertical real estate to show them, however, without risking the panel getting too tall and interfering with the top bar. I'm not sure how to reconcile in short order.

Reply:
Ah OK. The social media sharing options are only a convenience so I don't think it is essential to display them if we're short on space. It would be useful to have a comment in the code to indicate that why they are missing though.